airbyte-ci/connectors/regression-testing/regression_testing/backends/base_backend.py

```diff
@@ -0,0 +1,19 @@
+from abc import ABC, abstractmethod
+
+
+class BaseBackend(ABC):  # TODO
```

Comment:
I don't think I grasp the purpose of these backends, can you share more details?